sdk/cosmos/azure-cosmos-benchmark/pom.xml

```diff
@@ -294,19 +295,19 @@ Licensed under the MIT License.
       <!-- unit test -->
       <id>unit</id>
       <properties>
-        <env>default</env>
+
         <test.groups>unit</test.groups>
       </properties>
-      <activation>
-        <activeByDefault>true</activeByDefault>
-      </activation>
       <build>
         <plugins>
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-surefire-plugin</artifactId>
-            <version>3.5.2</version> <!-- {x-version-update;org.apache.maven.plugins:maven-surefire-plugin;external_dependency} -->
+            <artifactId>maven-failsafe-plugin</artifactId>
```

Comment:
for unit tests, would not surefire better? and also since there is no post failsafe stage,I guess would not be much difference?